### PR TITLE
report skipped tests

### PR DESCRIPTION
**Pull request by sechel**

### CODE REVIEW

**Comment by sechel** [comment]
And yes, I signed the CLA.

**Comment by sechel** [comment]
Hey @googlebot , can't you just merge this thing?

**Comment by jedateach** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "Daniel Herman <daniel.c.herman@gmail.com>",
     "Kristof Neirynck <k.neirynck@gmail.com>",
     "Marcin Szczepanski <mszczepanski@atlassian.com>",
-    "Sahat Yalkabov <sakhat@gmail.com>"
+    "Sahat Yalkabov <sakhat@gmail.com>",
+    "Stefan Sechelmann <stefan@sechel.de>"
```
Comment (dignifiedquire) [inline package.json]: Please remove this change this list is automatically updated on release

**Comment by dignifiedquire** [comment]
@sechel thanks, could you please change your commit message to follow our [convention](http://karma-runner.github.io/0.13/dev/git-commit-msg.html) and rebase onto latest `master` to resolve the merge conflicts.

**Comment by sechel** [comment]
Hey, is there something I missed doing to finish this up?

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Pull request merged**